_episodes/04-expertise.md

```diff
@@ -0,0 +1,270 @@
+---
```

Comment:
Note: the diff for this file cannot be viewed here due to episode renumbering. A diff is viewable in [this earlier PR in the sandbox.](https://github.com/data-lessons/instructor-training/pull/59/files)